cnf-certification-test/platform/suite.go

```diff
@@ -123,6 +123,11 @@ func testContainersFsDiff(env *provider.TestEnvironment) {
 	for _, cut := range env.Containers {
 		logrus.Debug(fmt.Sprintf("%s(%s) should not install new packages after starting", cut.Podname, cut.Data.Name))
 		debugPod := env.DebugPods[cut.NodeName]
+		if debugPod == nil {
+			ginkgo.Fail(fmt.Sprintf("Debug pod not found on Node: %s", cut.NodeName))
+			continue
```

Comment:
This could make the TC to be skipped in case of SNO. This is just a personal opinion, but I believe this TC must fail in case the debug pod of the node where this pod is running is nil. Debug pods are a pre-requisite for TNF to run properly.